libs/macos-cc-tools.manifest

```diff
@@ -6,5 +6,13 @@
     "algorithm": "sha512",
     "unpack": true,
     "filename": "MacOSX10.12.sdk.tar.xz"
```

Comment:
Do we need to keep the 10.12 SDK for any reason, or can we kill it and replace it with the new one?

Reply:
yeah I was thinking we might want to keep it around when I was testing both versions, but I think it makes sense to just remove this entirely to reduce confusion.